screens/RegisterScreen.tsx

```diff
@@ -73,13 +81,13 @@ const RegisterScreen = ({ navigation }: Props) => {
               style={authStyles.button}
               icon="login"
               mode="contained"
-              onPress={() => submit()}
+              onPress={() => handleRegister()}
```

Comment:
No need to create a "new" function since it doesn't take any parameters
```suggestion
              onPress={handleRegister}
```